src/Features/Core/Portable/PdbSourceDocument/PdbFileLocatorService.cs

```diff
@@ -126,6 +126,14 @@ internal sealed class PdbFileLocatorService : IPdbFileLocatorService
             }
 
             return result;
+
+            static FileStream? ReadFileIfExists(string fileName)
+            {
+                if (File.Exists(fileName))
```

Comment:
PEReader handled the FileNotFoundException fine, but the first chance exception is _very_ common, so easy enough to avoid it.